fail2ban/server/utils.py

```diff
@@ -146,10 +164,18 @@ def executeCmd(realCmd, timeout=60, shell=True, output=False, tout_kill_tree=Tru
 		"""
 		stdout = stderr = None
 		retcode = None
-		popen = None
+		popen = env = None
+		if varsDict:
+			if shell:
+				# build map as array of vars and command line array:
+				realCmd = Utils.buildShellCmd(realCmd, varsDict)
+			else: # pragma: no cover - currently unused
+				env = _merge_dicts(os.environ, varsDict)
```

Comment:
- what if there is an overlap? 
should may be all those get some unique prefix e.g. `_F2B_`? (yet to grasp the whole idea here)

- could those vars contain any symbol (e.g. `/`) not appropriate for env variables and thus should be sanitized first?